assets/js/components/ChecksResults/ChecksSelectionHints.jsx

```diff
@@ -36,7 +39,11 @@ function ChecksSelectionHints({ clusterId, selectedChecks }) {
             <Button
               className="bg-waterhole-blue mx-auto px-2 py-2 w-1/4 xs:w-full"
               onClick={() => {
-                navigate(`/clusters/${clusterId}/settings`);
+                navigate(
+                  usingNewChecksEngine
+                    ? `/clusters_new/${clusterId}/settings`
+                    : `/clusters/${clusterId}/settings`
+                );
```

Comment:
Here in this file, below, we need to add the `usingNewChecksEngine` to the `TriggerChecksExecutionRequest` component

Reply:
You mean to pass usingNewChecksEngine as prop to TriggerChecksExecutionRequest? 
Or something else ?

Reply:
Yes, exactly that